release_images.yml

```diff
@@ -92,19 +107,68 @@ release_images:
       example: False
       disable_sm_tag: False
       force_release: False
-  6:
-    framework: "huggingface_pytorch"
-    version: "1.10.2"
+  7:
+    framework: "tensorflow"
+    version: "2.9.2"
+    customer_type: "ec2"
     arch_type: "x86"
-    hf_transformers: "4.17.0"
-    inference:
-      device_types: [ "cpu", "gpu" ]
-      python_versions: [ "py38" ]
+    training:
+      device_types: ["cpu", "gpu"]
+      python_versions: ["py39"]
       os_version: "ubuntu20.04"
-      cuda_version: "cu113"
-      example: False        # [Default: False] Set to True to denote that this image is an Example image
-      disable_sm_tag: True # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
-      # to images being published.
-      force_release: False   # [Default: False] Set to True to force images to be published even if the same image
-      # has already been published. Re-released image will have minor version incremented by 1.
-  
+      cuda_version: "cu112"
+      example: False
+      disable_sm_tag: False
+      force_release: False
+  8:
+    framework: "tensorflow"
+    version: "2.9.2"
+    customer_type: "ec2"
+    arch_type: "x86"
+    training:
+      device_types: ["gpu"]
+      python_versions: ["py39"]
+      os_version: "ubuntu20.04"
+      cuda_version: "cu112"
+      example: True
+      disable_sm_tag: False
+      force_release: False
+  9:
+    framework: "tensorflow"
+    version: "2.9.2"
+    customer_type: "sagemaker"
+    arch_type: "x86"
+    training:
+      device_types: ["cpu", "gpu"]
+      python_versions: ["py39"]
+      os_version: "ubuntu20.04"
+      cuda_version: "cu112"
+      example: False
+      disable_sm_tag: False
+      force_release: False
+  10:
+    framework: "tensorflow"
+    version: "2.9.1"
+    customer_type: "ec2"
+    arch_type: "x86"
+    training:
+      device_types: ["hpu"]
+      python_versions: ["py38"]
+      synapseai_version: "1.5.0"
+      os_version: "ubuntu20.04"
+      example: False
+      disable_sm_tag: True
+      force_release: False
+  11:
```

Comment:
This is a duplicate, unless this is meant to be the example image for Habana TF2.9